vision_stack/buoy_detect/src/integration.cpp

```diff
@@ -116,8 +125,14 @@ void imageCallback(const sensor_msgs::ImageConstPtr& msg)
     {
         growhandle.start_grow(eroded, final_image, edgeMap, seed_points[i].x, seed_points[i].y, seed_points[i].color);
     }
+    
+    
     cv_ptr->image = final_image;
     final_pub->publish(cv_ptr->toImageMsg());
+    auto t4 = clk::now();
+    cout << "time re receive and queue : " << duration_cast<milliseconds>(t2 - t1).count() <<endl;
```

Comment:
@pranaypratyush @Nightfury18 Please use `ROS_DEBUG` and other logging functions that are provided by ROS, that makes setting log levels easy. It's hard to incorporate them once the code is in the master branch.